vision/module/location.py

```diff
@@ -49,7 +77,7 @@ def getDistance(self):
 
     ## Finding the Center
 
-    def getCenter(self):
+    def getCenter(self) -> tuple:
```

Comment:
You should add a parameter here that allows us to use previous calculations if in_frame returned true. For example, you could have a bool parameter that defaults to true called recalculate. Then you could check whether recalculate is true before getCenter() runs _getSlopes() and _groupSlopes() again.